compiler/src/iree/compiler/Dialect/Stream/Transforms/PackDispatchOperands.cpp

```diff
@@ -26,6 +26,17 @@ namespace IREE {
 namespace Stream {
 namespace {
 
+//===----------------------------------------------------------------------===//
+// Bitwidth utilities
+//===----------------------------------------------------------------------===//
+static int64_t getTypeBitwidth(Type type) {
```

Comment:
```suggestion
static int64_t getTypeBitWidth(Type type) {
```

For consistency we should match the MLIR naming (`getIntOrFloatBitWidth`)